### DISCUSSION TITLE
Usage with RCTText/RCTView

### DISCUSSION THREAD

**Discussion by tobimori** [jpudysz/react-native-unistyles#465] [Q&A]
Is there a way to use RCTText / RCTView directly with Unistyles? The default Text component in RN seems to be very imperformant, so this would be a solution worth trying, if it works with Unistyles.

ref https://x.com/hirbod_dev/status/1850526592570695942

**Comment by jpudysz** [comment]
Yup, I have it on my todo list but for `beta.6`

**Reply by jpudysz** [reply]
Issue for reference: https://github.com/jpudysz/react-native-unistyles/issues/461
